tests/main/interfaces-hardware-control/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Ensure that the hardware-control interface works.
+
+summary: |
+    The hardware-control interface allows a snap to control hardware and platform devices.
+
+    A snap which defines the hardware-control plug must be shown in the interfaces list.
+    The plug must not be connected on install and, as usual, must be able to be
+    reconnected.
+
+    A snap declaring a plug on this interface must be able to write files in /sys/{block,bus,class,devices}
+
+prepare: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+
+    echo "Given a snap declaring a plug on the hardware-control interface is installed"
+    install_local hardware-control-consumer
```

Comment:
Did you forget to "git add" the associated files?

Reply:
I did!